NOTES.md

```diff
@@ -1,3 +1,186 @@
+# Release 2.0.0-RC1 (Musashi)
+
+Released on 2015-03-23
+
+This is a release candidate for the upcoming RethinkDB 2.0 release.
+It is not for production use. It might still have bugs. If you decide
+to test it, please [back up all][backup-docs] your data first.
+
+Release highlights:
+* Support for attaching a changefeed to the `get_all` and `union` commands
+* Improved support for asynchronous queries
+* Added a new feature called `tables` which can be used to store JSON documents
+
+Read the [blog post][2.0-RC1-blog] for more details.
+
+[2.0-RC1-blog]: http://rethinkdb.com/blog/2.0-release-candidate/
+
+## Compatibility ##
+
+This release may not be compatible with RethinkDB 2.0.
+
+### Handling of negative zero in indexes ###
+
+IEEE 754 floating point numbers distinguish between negative (-0) and
+positive (+0) zero.  The following information is only relevant if you
+are storing negative zero values in your documents.  We expect very
+few users to be affected by this change.
+
+ReQL compares -0 and +0 as equal in accordance with IEEE 754.  In
+previous versions of RethinkDB, -0 and +0 were however treated as
+distinct values in primary and secondary indexes.  This could lead to
+inconsistent behavior and wrong query results in some rare cases.
+
+Starting with RethinkDB 2.0, -0 and +0 are indexed as equal values.
+Secondary indexes can be [migrated online][1.14-outdated-index] using
+the `rethinkdb index-rebuild` utility.  If any of your documents have
+negative zero values in their primary keys, those documents will
+become partially inaccessible in RethinkDB 2.0. You will need to
+re-import the affected tables using the `rethinkdb dump` and
+`rethinkdb restore` commands. See [Back up your data][backup-docs] for
+more information.
+
+If you are unsure if any of your documents are affected, you can run
+`python -m rethinkdb._negative_zero_check` after upgrading both the
+server and Python driver.  See the output of `python -m
+rethinkdb._negative_zero_check --help` for additional options.
+
+[1.14-outdated-index]: http://rethinkdb.com/docs/troubleshooting/#my-secondary-index-is-outdated
+[backup-docs]: http://www.rethinkdb.com/docs/backup/
+
+### API-breaking changes ###
+
+* `between` no longer accepts `null` bounds. The new `r.minval` and
+  `r.maxval` can be used instead
+
+* The `any` and `all` commands have been removed. The `or` and `and`
+  commands can be used instead
+
+* `indexes_of` has been renamed to `offsets_of`
+
+* The `squash` argument to `changes` now defaults to `false`
+
+* The type hierarchy for exception types in the Python driver changed.
+  All exceptions including `RqlDriverError` now inherit from the
+  `RqlError` type.  `RqlRuntimeError`, `RqlCompileError` and
+  `RqlClientError` additionally inherit from the new `RqlQueryError`
+  type.
```

Comment:
There should be no period after the last senctence (replaces my previous comment about adding periods to the other points)